.travis.yml

```diff
@@ -0,0 +1,48 @@
+language: ruby
+cache: bundler
+dist: trusty
+sudo: required
+
+rvm:
+- 2.3.7
+- 2.4.4
+- 2.5.1
+
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install python
+  - nvm install node
+  - nvm use node
+
+install:
+  - pip install --user bumpversion
+  - npm install -g semantic-release
+  - npm install -g @semantic-release/exec
+  - npm install -g @semantic-release/git
+  - bundle install
+
+script:
+  - bundle exec rake
+
+before_deploy:
+  - bundle exec rake test:appveyor_status
+
+deploy:
+  - provider: script
+    script: npx semantic-release
+    on:
+      branch: master
+      rvm: 2.5.1
+
+  - provider: rubygems
+    api_key: $RUBYGEMS_API_KEY
+    gem: ibm_core_cloud_sdk
```

Comment:
I think this should be ibm_cloud_sdk_core to be consistent with the python and node packages perhaps?

Reply:
good catch! I don't know why I did that. thanks!